### PR TITLE
MP#5 setup product page by junyanwang

### PR DESCRIPTION
**Pull request by deborahwang**

### CODE REVIEW

File: companies/junyanwang/A Frozen SingAlong Celebration.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+   <!-- Page Title
+   ================================================== -->
+   <section id="page-title">	
+   	  
+		<div class="row">
+
+			<div class="twelve columns">
+
+				<h1>Kun Su<span></span></h1>
```
Comment (KunSu) [inline companies/junyanwang/A Frozen SingAlong Celebration.php]: let's change the page title to your own name and tags for all 10 pages.

File: index.php [hunk 2]
```diff
@@ -273,6 +273,136 @@
                             </a>
                         </div>
                     </div> <!-- item end -->
+
```
Comment (KunSu) [inline index.php]: You should add a new section here. Check out this as reference https://github.com/KunSu/CMPE272MarketPlace/pull/20/files#diff-828e0013b8f3bc1bb22b4f57172b019dR290

**Review by KunSu — COMMENTED** [review]
Looks good for me. Just need some small changes.

**Comment by KunSu** [comment]
You might need to rebase master since Tayor's PR is already merged. It means whenever other teammates' PR merged into the master, you need to perform a rebase (apply your changes on top of the master). If the auto-merge button is green and there is no conflict, you don't need to do so.

**Comment by KunSu** [comment]
And try to resize all your images to the same width and height if you have time.
image

File: companies/junyanwang/A Frozen SingAlong Celebration.php (hunk 1 above)
Comment (deborahwang) [reply companies/junyanwang/A Frozen SingAlong Celebration.php]: Updated.

File: index.php (hunk 2 above)
Comment (deborahwang) [reply index.php]: Done.

**Review by KunSu — APPROVED** [review]
LGTM

**Pull request merged**